runelite-client/src/main/java/net/runelite/client/config/RuneLiteConfig.java

```diff
@@ -116,12 +116,12 @@ default boolean requestFocusOnNotification()
 	}
 
 	@ConfigItem(
-		keyName = "useSmallFont",
-		name = "Small font in dynamic overlays",
-		description = "Toggles between small and regular RuneScape font for in-game overlays"
+			keyName = "fontType",
```

Comment:
Fix your indentation (in Intellij Idea find continuation indent and change it to 2)